click_odoo_contrib/update.py

```diff
@@ -113,10 +220,26 @@ def OdooEnvironmentWithUpdate(database, ctx, **kwargs):
 @click.option(
     "--if-exists", is_flag=True, help="Don't report error if database doesn't exist"
 )
-def main(env, i18n_overwrite, update_all, if_exists):
+@click.option(
+    "--watcher-beat",
+    default=10.0,
+    type=float,
+    help="Time to wait between DB lock watcher loops",
+)
+@click.option(
+    "--watcher-max-seconds",
+    default=30.0,
+    type=float,
+    help="Max DB lock seconds allowed before aborting the update process. 0=disable",
```

Comment:
You can skip spawning the watcher by setting `--watcher-max-seconds=0`.

I think that to leave the watcher on by default is safer since it covers more use cases, but I left the option to skip it so you can feel safer about it. :blush:

Reply:
I insist to have the watcher disabled by default.

Reply:
Done.